src/current/v23.2/work-with-virtual-clusters.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Work with virtual clusters
+summary: Learn how to connect to and work with virtual clusters.
+toc: true
+docs_area: deploy
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Refer to the [Cluster Virtualization Overview]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}#known-limitations) for further detail.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} This page shows how to work with a cluster with [cluster virtualization]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}).
```

Comment:
```suggestion
{% include_cached new-in.html version="v23.2" %} This page is a guide to working with a CockroachDB cluster with [cluster virtualization]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}) enabled.
```

Reply:
Not sure of the exact language here, but defining how we will talk about **the** cluster when also talking about virtual clusters feels important. I assume this sentence meant the overarching CockroachDB cluster.

Reply:
Yeah, I think that way back in the beginning it was decided to refer to this as "with/without cluster virtualization enabled" so I went with that. Agree it is wordy and we should discuss with PM and Eng.